fluentd/Dockerfile

```diff
@@ -49,7 +49,8 @@ ADD configs.d/ /etc/fluent/configs.d/
 ADD filter_k8s_meta_for_mux_client.rb /etc/fluent/plugin/
 ADD out_syslog_buffered.rb out_syslog.rb out_rawtcp.rb /etc/fluent/plugin/
 ADD parser_viaq_docker_audit.rb viaq_docker_audit.rb /etc/fluent/plugin/
-ADD run.sh generate_throttle_configs.rb generate_syslog_config.rb ${HOME}/
+ADD run.sh generate_syslog_config.rb ${HOME}/
+ADD lib/*/lib/*.rb ${HOME}/
```

Comment:
What is this adding? I don't see any `lib/*/lib/` directories.

Reply:
Its this file move from root: https://github.com/openshift/origin-aggregated-logging/pull/1183/files#diff-0ae11326c532c900e8e799ba61a2bec6

Reply:
Thanks, I didn't look close enough at the destination paths I guess